### ISSUE TITLE
NEAR and RECENT are being searched for as terms instead of affecting the search results

### ISSUE THREAD

**Issue by songproducer** [yacy/yacy_search_server#235]
Not sure if the docs for NEAR and RECENT here are out of date:
http://www.yacy-websuche.de/wiki/index.php/En:SearchParameters

**Comment by luccioman** [comment]
Yes @songproducer that wiki page appears to be out of date regarding the   `NEAR`, `RECENT` and `LANGUAGE:` search parameters.
Instead, as detailed in your peer index.html page "more options..." section, you can use the `/near`, `/date` and `/language/<lang>` parameters. For information, the change seems to date back to commit cc6499bf8d67800c24e9e23d7d1442dedfcd7bbc.

Would you like to update the wiki page?

**Comment by songproducer** [comment]
@luccioman I tried to sign up to the wiki but everything is in German and I got some error that I translated to mean something about cookies not being enabled (although my cookies are enabled). Here is the error message:```

 bei der Anmeldung
Das Benutzerkonto wurde nicht erstellt, da die Datenherkunft nicht ermittelt werden konnte. Es muss sichergestellt sein, dass Cookies aktiviert sind. Danach diese Seite erneut laden und es wieder versuchen
```

**Comment by luccioman** [comment]
Thanks for reporting this @songproducer, I also just had the same issue with various major browsers when trying to authenticate on YaCy Wiki. I notified @Orbiter , I hope he could find some time to fix this.
